package.json

```diff
@@ -44,7 +44,7 @@
 		"morgan": "1.7",
 		"multer": "1.2",
 		"multipipe": "1.0",
-		"nodemailer": "2.6",
```

Comment:
Yes, that's what I was suggesting. Good to know! Looks like the change from MIT for 2.x to EUPL license for 3.x is still fine for us too ( https://en.wikipedia.org/wiki/European_Union_Public_Licence )